.env

```diff
@@ -1,2 +1,7 @@
 REACT_APP_API_STUDY_SERVER=api/study-server
 REACT_APP_API_CASE_SERVER=api/case-server
+
+REACT_APP_CLIENT_ID=#
```

Comment:
It would be nice if our build produced html and javascript that work anywhere. The specific parts should come from kubernetes configmaps (or equivalently docker mounts "-v"). This means we can't have these new values here. Please try to implement it like that